backend/typescript/server.ts

```diff
@@ -2,6 +2,8 @@ import cookieParser from "cookie-parser";
 import cors from "cors";
 import express from "express";
 import * as firebaseAdmin from "firebase-admin";
+import swaggerUi from 'swagger-ui-express';
```

Comment:
Small nitpick: can we change to double quotes? Same thing below, could run `yarn fix`.